packages/kuma-gui/src/app/connections/data/index.spec.ts

```diff
@@ -59,7 +65,21 @@ describe('ConnectionCollection', () => {
               type: '',
               zone: '',
             },
-            tcp: {},
+            tcp: {
+              ssl: {
+                certificate: {
+                  'spiffe://default': {
+                    'local-zone': {
+                      mesh: {
+                        'local/ns/kuma-demo/sa/default': {
+                          expiration_unix_time_seconds: 1765820805,
+                        },
```

Comment:
How difficult would it be to do the same thing we do with ip address here?

i.e. for ips we end up with `obj['127.0.0.1'].stuff.here = 0`

would be good if we could end up with:

```
obj['spiffe://default.local-zone.mesh.local/ns/kuma-demo/sa/default'].stuff.here = 0
```

I would guess anything with `://` in it should be treated like this (`://` is impossible to add as a user so its safe to assume its some sort of uri added by internal means)

Reply:
Ok so we had a little pair on this and it looks like its not currently possible because Kubernetes ServiceAccounts can have `.`s in them which means we could have a spiffe uri like:

`spiffe://default.local-zone.mesh.local/ns/kuma-demo/sa/default.stuff`

Currently its impossible for us to know at what point the spiffe uri ends.

Lets chat with backend about possibility of escaping `.`s  somehow, if we are using `.`s as separators then any values within the separators should have their `.`s escaped.

Reply:
Just to share thoughts: The other thing I've noticed is (which might not be true), if the `spiffe://` is always the second to last segment we could use that as the rule to parse out the spiffe as a full uri.